airflow/build/airflow_worker_process.cfg

```diff
@@ -87,7 +87,7 @@ parallelism = 32
 dag_concurrency = 16
 
 # Are DAGs paused by default at creation
-dags_are_paused_at_creation = True
+dags_are_paused_at_creation = False
```

Comment:
Does this mean we don't need to manually unpause dags anymore? then we can probably delete the logic [here](https://github.com/Open-EO/openeo-openshift-driver/blob/2e35c86e3a2b6a73eae4a56286470bc1e05eabf0/services/jobs/jobs/dependencies/airflow_conn.py#L31)

Reply:
agree, I updated the logic